conf/messages.en

```diff
@@ -97,4 +97,10 @@ primaryContactDetails.fullNamePage.lastName.error.specialCharacters = Enter a la
 
 sessionTimout.title = For your security, we signed you out
 sessionTimout.paragraph.saved = We saved your answers.
-sessionTimout.signin.button = Sign in
\ No newline at end of file
+sessionTimout.signin.button = Sign in
+
+primaryContactDetails.sectionHeader = Primary contact details
```

Comment:
Please delete:
```
primaryContactDetails.sectionHeader = Primary contact details
```

This already exists.

Reply:
done